apps/mi/mi-adf-shir/test.yaml

```diff
@@ -8,6 +8,7 @@ spec:
   values:
     image: sdshmctspublic.azurecr.io/mi/adf-integration-runtime:prod-0641eb6-20221206112502 #{"$imagepolicy": "flux-system:mi-adf-shir"}
     replicaCount: 2
+    memoryLimits: '4096Mi'
```

Comment:
Increasing this as we are seeing a lot of OOM errors, increasing this seems to have reduced it, but may need more confirmation.